ros/src/visualization/rqt_basic.perspective

```diff
@@ -239,41 +239,45 @@
               "groups": {
                 "history": {
                   "keys": {
+                    "config_9": {
+                      "type": "repr",
+                      "repr": "u'file:///mnt/hgfs/share/fusionx/CarND-Capstone/ros/src/visualization/rqt_multiplot_basic.xml'"
```

Comment:
The file `rqt_multiplot_basic.xml` is being renamed to `rqt_multiplot_simulation.xml`. Should this (and a few other locations in this file) reflect that?

Reply:
Unfortunately the RQT perspective file saves the full path to the last multiplot layout I loaded, but this won’t be valid for anyone else anyways.  I don’t know how to disable this and don’t want to manually edit the exported file.  I think when this perspective is imported, it will just have a blank multiplot panel since the path isn’t valid, but then you can open the multiplot file manually and then it will be saved for your system.

Reply:
I could re-export this file after loading the simulation multiplot layout file, but I think it still won’t be a valid path for anyone else.